azure/util.py

```diff
@@ -0,0 +1,325 @@
+import argparse
+import yaml
+import json
+import os
+from os import walk
+import os.path
+import shutil
+import subprocess
+import uuid
+import urllib.request
+import tarfile
+import zipfile
+from shutil import copyfile
+import logging
+
+PETCTL_DIR = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+logging.basicConfig(format='%(asctime)s - %(levelname)s - %(name)s -   %(message)s',
+                    datefmt='%m/%d/%Y %H:%M:%S',
+                    level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+# This method runs all commands in a separate
+# process and returns the output
+def run_commands(cmds):
+    output = []
+    set_kubeconfig_environment_var()
+
+    for cmd in cmds:
+        process = subprocess.run(
+            cmd,
+            shell=True,
```

Comment:
Nice! Thanks for making this change. Just a couple of minor comments:
1. `shell=True` - is this needed? 
2. `output` is unused. Perhaps you meant to append the result of `subprocess.run` to it and return it?
3.  What happens if `subprocess.run` throws? Currently, this will fail all subsequent commands but if you think it's appropriate, you can surround `subprocess.run` in a `try..except..finally` and ignore errors. 
4. Along the same lines, you might want to pass `check_output=True`

Reply:
1. Using` shell=True` to avoid splitting the commands and using the string queries as-is. Splitting can lead to small errors.
2. Thanks for catching this, will remove it.
3. We need to run the commands in order so if it throws and error on the first command we don't want to ignore the error.
4. I think check_output is equivalent to process.stdout that I'm using. Please correct me if this is wrong.

Reply:
1., 2. and 3.: thanks!
4. Sorry, I was confused with some of the parameters. There is a `check` option (not `check_output`) that throws an exception if the process returns a non-zero error code.   Given what you said in 3., you might want to pass `check=True` and fail fast if one of the commands fails.

Reply:
Got it. Made the change, thanks for the suggestion!